test/chat_api/slack_test.exs

```diff
@@ -87,11 +87,40 @@ defmodule ChatApi.SlackTest do
       customer: customer,
       thread: thread
     } do
-      channel = thread.slack_channel
       text = "Hello world"
+      customer_email = "*Email:*\n#{customer.email}"
+      channel = thread.slack_channel
 
       assert %{
-               "blocks" => _blocks,
+               "blocks" => [
+                 %{
+                   "text" => %{
+                     "text" => ^text
+                   }
+                 },
+                 %{
+                   "fields" => [
+                     %{
+                       "text" => "*Name:*\nAnonymous User"
+                     },
+                     %{
+                       "text" => ^customer_email
+                     },
+                     %{
+                       "text" => "*URL:*\nN/A"
+                     },
+                     %{
+                       "text" => "*Browser:*\nN/A"
+                     },
+                     %{
+                       "text" => "*OS:*\nN/A"
+                     },
+                     %{
+                       "text" => "*Timezone:*\nN/A"
```

Comment:
nice! can we also add a test where these values actually exist on the customer?

(you can create a test customer with `customer_fixture(account, %{name: "...", time_zone: "...", etc: "etc"})`)

Reply:
@reichert621 do you think that I should write another function or improve the one above with actual values (in `get_message_payload/2 returns payload for initial slack thread` test)?

Reply:
both would be nice! that way we test that the fallback values (e.g. "N/A") are working properly :)